scripts/setup_test.py

```diff
@@ -388,17 +410,19 @@ def mock_check_call(cmd_tokens):
                     common.YARN_VERSION, common.YARN_VERSION)])
         self.assertEqual(all_cmd_tokens, ['./configure', 'make'])
 
-    def test_package_install_with_windows_x86(self):
-        def mock_exists(unused_path):
+    def test_package_install_with_windows_x86(self) -> None:
+        def mock_exists(unused_path: str) -> Literal[False]:
             return False
 
-        def mock_url_retrieve(url, filename): # pylint: disable=unused-argument
+        def mock_url_retrieve(url: str, filename: str) -> None: # pylint: disable=unused-argument
             self.urls.append(url)
 
-        def mock_check_call(commands):
-            mock_check_call.commands = commands
+        def mock_check_call(commands: List[str]) -> None:
+            # Here commands attribute is defined but still mypy is unable to
+            # detect this. Thus, added an ignore.
+            mock_check_call.commands = commands  # type: ignore[attr-defined]
```

Comment:
Why?

Reply:
Here `commands` is defined in `mock_check_call`  but still mypy can't see this. that's why I added this ignore. does it look good to you?

Reply:
Why MyPy can't see it, did you try to investigate?